scene/3d/visual_instance.cpp

```diff
@@ -60,7 +60,7 @@ void VisualInstance::_notification(int p_what) {
 			if (skeleton)
 				VisualServer::get_singleton()->instance_attach_skeleton( instance, skeleton->get_skeleton() );
 			*/
-
+			ERR_FAIL_COND(get_world().is_null());
```

Comment:
The fix seems fine, but in what case do we receive `NOTIFICATION_ENTER_WORLD` without a valid `get_world()`?

Reply:
I don't know exactly but somehow the function of `Ref<World> Viewport::find_world() const` returns _else_ situation. Maybe it returns `Ref<World>()` if `is_inside_world()` is _false_ or `data.viewport` is `null`. How can we test that `get_world()` why returns _null_ or _empty_ with a better way? What is your opinion?